torch/fx/passes/backends/cudagraphs.py

```diff
@@ -0,0 +1,53 @@
+import torch
+from torch.fx.passes.infra.partitioner import CapabilityBasedPartitioner
+from torch.fx.passes.operator_support import OperatorSupport
+from torch.fx.passes.tools_common import CALLABLE_NODE_OPS
+from torch.fx.passes.fake_tensor_prop import FakeTensorProp
+from torch.utils._pytree import tree_map
```

Comment:
curious how come `_pytree` is considered an internal only

Reply:
We should make it public; we weren't sure if we had stabilized the API enough when we originally added it. cc @zou3519